index.html

```diff
@@ -1,10 +1,8 @@
-<html>
 
+<html>
 <head>
-	<script src="${pluginsBaseRootUrl}/web.shared/jquery.js" type="text/javascript"></script>
-	<script src="${pluginsBaseRootUrl}/web.shared/jquery.class.js" type="text/javascript"></script>
-	<script src="${pluginsBaseRootUrl}/web.shared/jquery.cookie.js" type="text/javascript"></script>
-	<script src="${pluginsBaseRootUrl}/web.shared/elvis_api/js/jquery.elvis.js" type="text/javascript"></script>
+    <script src="assets-client-sdk.min.js"></script>
```

Comment:
Should this include a version number? Now it's not really clear if this is outdated or not. Maybe we should publish it to a CDN like cdnjs?

Reply:
Good ideas 👍

Reply:
Used unpkg.com CDN to auto-load latest client sdk directly from the published NPM package. Tested with latest version of the sdk, works fine.